torch/csrc/autograd/function.h

```diff
@@ -23,6 +23,11 @@ using function_list = std::vector<std::pair<std::shared_ptr<Function>, int>>;
 
 // State used to create "backward" functions
 struct FunctionFlags {
+  // Roughly speaking, is_executable corresponds to requires_grad.
+  // See http://pytorch.org/docs/notes/autograd.html for more details:
+  // roughly speaking, both is_executable and is_volatile specify whether or
+  // not backwards gradient computation will be performed for a function,
+  // but they differ in their precedence.
```

Comment:
roughly speaking is repeated twice. Also, `is_volatile` is never saved in the function object. It's just a helper for `wrap_output` functions